eventlogger/src/test/java/gov/cms/ab2d/eventlogger/reports/sql/LoggerEventSummaryTest.java

```diff
@@ -51,7 +51,7 @@ void getSummaryBasic() {
         LoggableEvent e4 = new FileEvent(usr, jobId, file1, FileEvent.FileStatus.OPEN);
         e4.setTimeOfEvent(firstTime.plusDays(3));
         logger.log(e4);
-        LoggableEvent e5 = new ContractSearchEvent(usr, jobId, "CONTRACT1", 100, 100, 100, 2, 1000, 1000, 1);
+        LoggableEvent e5 = new ContractSearchEvent(usr, jobId, "CONTRACT1", 100, 100, 100, 2, 100, 1000, 1000, 1);
```

Comment:
Reusing the value of 100 open up a cross-wire vulnerabilty i.e. if one has two fields a and b and sets them both to the same value, then one can not detect if b was updated when a should be.  

Consider using unique values for each member field.

Reply:
agread